.gitlab/lassen-jobs.yml

```diff
@@ -78,7 +78,7 @@ lassen_gcc_cuda_tpls:
       - COMPILER_SPEC: gcc@8.3.1
         CUDA_SPEC: [cuda@11.5.0]
   variables:
-    SPEC: "%${COMPILER_SPEC} cstd=99 cxxstd=14 precision=double ~int64 +mpi+openmp+cuda+raja+magma+superlu-dist cuda_arch=70 ^superlu-dist+cuda cuda_arch=70 ^magma+cuda cuda_arch=70 ^raja+cuda~openmp~examples~exercises cuda_arch=70 ^${CUDA_SPEC}"
+    SPEC: "%${COMPILER_SPEC} cstd=99 cxxstd=14 precision=double ~int64 +mpi+openmp+cuda+raja+magma+superlu-dist+petsc+hypre+ginkgo cuda_arch=70 ^ginkgo+cuda cuda_arch=70 ^hypre ^petsc+cuda cuda_arch=70 ^superlu-dist+cuda cuda_arch=70 ^magma+cuda cuda_arch=70 ^raja+cuda~openmp~examples~exercises cuda_arch=70 ^${CUDA_SPEC}"
```

Comment:
Maybe this should be `^petsc` rather than `^petsc+cuda cuda_arch=70` (I'm not sure cuda capabilities in PETSc are actually getting used)

Reply:
Even though we dont use CUDA in PETSc, we should keep `petsc+cuda` since this is how xsdk will build it.

Reply:
Also ginkgo is being added in a separate PR woth kokkos.